avm/res/storage/storage-account/README.md

```diff
@@ -2052,13 +2066,6 @@ Networks ACLs, this value contains IPs to whitelist and/or Subnet information. I
 
 - Required: No
 - Type: object
-- Default:
```

Comment:
That's a bit of a suprising one we should double-check

Reply:
@AlexanderSehr that happened in another recently merged PR for input parameters made null able, ref https://github.com/Azure/bicep-registry-modules/pull/1474/files#r1545613784

Reply:
Hey @eriqua, not sure I follow :o

Reply:
I thought you were commenting on the removed Default line. Otherwise I didn't get your comment

Reply:
I did @eriqua. The but while in the PR you linked defaults were updated, the above is a different change. And as per the comment below it was seemingly implemented as part of this PR to fix an issue. I'll resolve this comment as I only created it because I saw & commented the readme change before I saw the code change. As that one is the actual root cause, I suggest we only discuss the matter there.